.appveyor.yml

```diff
@@ -23,7 +23,7 @@ install:
   - conda install gtest cmake -c conda-forge
   # Testing
   - conda install r r-rcpp r-base r-devtools -c r
-  - conda install xtensor==0.10.9 -c conda-forge
+  - conda install xtensor==0.10.10 -c conda-forge
```

Comment:
This is 0.10.10 and for travis it's 0.10.11? On purpose?

Reply:
Nope, should be 0.10.11. (Although it is probably not really important)